docs/buildspecs/e4s_testsuite.rst

```diff
@@ -0,0 +1,49 @@
+.. Note:: Please see :ref:`tutorial_setup` before you proceed with this section
+
+.. _buildtest_e4s_testsuite:
+
+Buildtest E4S Testsuite Integration
+===================================
+
+The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
+software products provided by `E4S <https://e4s.io>`_ deployments. 
+E4S is a spack based software stack but because not all spack packages provide internal testing functionality it 
+is useful to have an alternative framework for testing deployed software.
+
+The E4S Testsuite operates on a hierarchy of shell scripts with each test containing its own setup, clean, compile 
+and run scripts. The `top level driver script <https://github.com/E4S-Project/testsuite/blob/master/test-all.sh>`_ sets up the test environment based on a configuration file which sets 
+compilers, mpi run commands and other environment specific parameters. It then iterates through the selected tests 
+and generates output consisting of log files for individual test runs and a summary listing of each test’s success or failure. The `README <https://github.com/E4S-Project/testsuite/blob/master/README.md>`_ provides more detail on its structure and usage.
+
+Tests in the testsuite typically include source code and build files needed to build a small test application against 
+libraries provided by the spack install along with any input files required for the subsequent run test. These are 
+often extracted from test functionality included in the product’s source 
```

Comment:
I think you can remove this sentence `These are often extracted from ...` User is not going to care about this detail